app/tasks/analyze.py

```
@@ -38,4 +267,534 @@ async def _run() -> dict:
+                    meta={
+                        'current': index,
+                        'total': total_count,
+                        'status': f'{coin_name} 분석 중... ({index}/{total_count})',
+                        'current_coin': coin_name,
+                        'percentage': int((index / total_count) * 100)
+                    }
+                )
+
+                # 개별 코인 분석
+                try:
+                    result, model = await analyzer.analyze_coins_json([coin_name])
```

Ruff (0.14.3)

Unpacked variable `result` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)